plugin/main.cpp

```diff
@@ -19,7 +19,8 @@
 Q_DECL_EXPORT int main(int argc, char *argv[])
 {
     QLoggingCategory::setFilterRules(QStringLiteral("geoclue.provider.mlsdb.debug=false\n"
-                                                    "geoclue.provider.mlsdb.position.debug=false"));
+                                                    "geoclue.provider.mlsdb.position.debug=false\n"
+                                                    "geoclue.provider.mlsdb.online.debug=false"));
```

Comment:
Ah, thanks!  I had not known that about the two-param version.  Do you think we should change them to use QtWarningMsg as the default filtering level?